transcendence-app/src/chat/chat.controller.ts

```diff
@@ -80,6 +80,7 @@ export class ChatController {
   @ApiCreatedResponse({ description: 'Create a chatroom', type: Chatroom })
   @ApiBadRequestResponse({ description: 'Bad Request' })
   @ApiUnprocessableEntityResponse({ description: 'Unprocessable entity' })
+  @ApiServiceUnavailableResponse({ description: 'Service Unavailable' })
```

Comment:
we should probably recreate the swagger spec after this change

Reply:
Updated. Thank you for your suggestion.